src/draw/lv_image_decoder.c

```diff
@@ -94,10 +94,10 @@ lv_result_t lv_image_decoder_get_info(const void * src, lv_image_header_t * head
     }
 
     lv_result_t res = LV_RESULT_INVALID;
-    lv_image_decoder_t * d;
-    _LV_LL_READ(img_decoder_ll_p, d) {
-        if(d->info_cb) {
-            res = d->info_cb(d, src, header);
+    lv_image_decoder_t * decoder;
```

Comment:
Is there a particular reason for this change? I'm ok with that, just want to be sure if it's really just a minor refactoring.

Reply:
No specific reason, I just like to follow naming conventions, and clean code... meaning: avoid short (1-2 letters) names and use descriptive (full length) names.